src/bin/miri.rs

```diff
@@ -487,6 +487,13 @@ fn main() {
                 "full" => BacktraceStyle::Full,
                 _ => panic!("-Zmiri-backtrace may only be 0, 1, or full"),
             };
+        } else if let Some(param) = arg.strip_prefix("-Zmiri-external_c_so_file=") {
+            let filename = param.to_string();
+            if std::path::Path::new(&filename).exists() {
+                miri_config.external_c_so_file = Some(filename);
```

Comment:
Since we currently only allow one to be set (this is fine for now, we can add more later) maybe we should error out if `external_c_so_file` is already set to avoid confusion?

Reply:
good idea